packages/block-library/src/columns/style.scss

```diff
@@ -10,7 +10,7 @@
 		flex-wrap: nowrap;
 	}
 
-	&.has-background {
+	&:where(.has-background) {
```

Comment:
Just checking something. When I set padding in my theme.json via styles

```json
	"styles": {
		"spacing": {
			"padding": "55px",
			"blockGap": "66px"

		},
 ...
```

These styles override the body styles of `padding: 55px`. Block spacing works as expected.

Reply:
Good callout. This is correct as I understand it -- these styles will override top-level (body) styles from `theme.json`, but not `theme.json` styles for the Columns block _specifically_, or of course values from global styles/the block controls. I confirmed this is how the Group block behaves as well.